tests/Server/Workspace/SymbolTest.php

```diff
@@ -58,9 +58,10 @@ public function testEmptyQueryReturnsAllSymbols()
             new SymbolInformation('TestInterface',      SymbolKind::INTERFACE,   $this->getDefinitionLocation('TestInterface'),                                ''),
             new SymbolInformation('test_function',      SymbolKind::FUNCTION,    $this->getDefinitionLocation('test_function()'),                              ''),
             new SymbolInformation('ChildClass',         SymbolKind::CLASS_,      $this->getDefinitionLocation('ChildClass'),                                   ''),
+            new SymbolInformation('define',             SymbolKind::CONSTANT,    $this->getDefinitionLocation('TEST_PROPERTY'),                                ''),
```

Comment:
If this is the defined constant, shouldn't the symbol name be `TEST_PROPERTY` (or `TEST_DEFINE_CONSTANT`), not `define`?